src/lmic/radio.c

```diff
@@ -534,7 +534,7 @@ static void configLoraModem () {
 
         // set ModemConfig2 (sf, AgcAutoOn=1 SymbTimeoutHi)
         u1_t mc2;
-        mc2 = (SX1272_MC2_SF7 + ((sf-1)<<4)) | 0x04 | ((LMIC.rxsyms >> 8) & 0x3));
+        mc2 = ((SX1272_MC2_SF7 + ((sf-1)<<4)) | 0x04 | ((LMIC.rxsyms >> 8) & 0x3));
```

Comment:
Huh. I wonder how this got through CI testing, as I thought I was compiling for SX1272.

This will work, but I think it's more matching the rest of the style to remove the extra `)` at the end of the line, i.e.:

```c
mc2 = (SX1272_MC2_SF7 + ((sf-1)<<4)) | 0x04 | ((LMIC.rxsyms >> 8) & 0x3);
```

The intent is to compute `a | b | c`; the extra parens make it harder to read.

Reply:
I don't know why I put an extra `(` instead of deleting the `)` 😄